<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register_Page</class>
 <widget class="QWidget" name="Register_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Register_Page  QPushButton
{
	background-color: rgb(85, 0, 255);
	border-radius:6px;
}
QWidget#Register_Page  QPushButton:hover
{
	background-color: rgb(255, 0, 255);
}
QWidget#Register_Page
{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneNum">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>联系电话</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="userNa">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Pwd">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="placeholderText">
    <string>请输入联系电话</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="adrEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入住址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>欢迎注册！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Address">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>住址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
